### PR TITLE
Attributes Editor

### PR DESCRIPTION
**Pull request by deanblacc**
### Demo

https://user-images.githubusercontent.com/4340994/169579467-e4de0dfc-37e0-436a-a529-4429d76c0f10.mov

### CODE REVIEW

File: packages/gui/src/components/html/editor.tsx [hunk 1]
```diff
@@ -2,11 +2,27 @@ import { Editor } from '../Editor'
...
-import { isNil, values } from 'lodash-es'
+import { filter, isNil, values } from 'lodash-es'
 import IconButton from '../ui/IconButton'
 import { Trash2 } from 'react-feather'
-import { Label } from '../primitives'
+import { Label, Combobox } from '../primitives'
 import { SelectInput } from '../inputs/SelectInput'
+import { AttributeEditor } from './AttributeEditor'
+
+const HTML_TAGS = [
```
Comment (johno) [inline packages/gui/src/components/html/editor.tsx:12]: We may as well add all the attributes here in the data dir.

**Review by johno — APPROVED** [review]
One comment, aside from that lgtm! 🎉

**Comment by mrmrs** [comment]
Super rad. Thinking we should get consistent on x vs trashcan. I can make that change.

**Comment by mrmrs** [comment]
After giving it a run through, I think we'll want to add the attributes below or above the control. Right now they get added in between the label and the select.

File: packages/gui/src/components/html/AttributeEditor.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        return (
+          <div sx={{ display: 'flex', alignItems: 'center', gap: '.5rem' }}>
+            <Label>
+              {key}
+              <input
+                value={attrValue}
+                onChange={(e) => onChange({ ...value, [key]: e.target.value })}
+              />
+            </Label>
+            <IconButton>
```
Comment (mrmrs) [inline packages/gui/src/components/html/AttributeEditor.tsx:74]: Is this missing an action? Removing attributes isn't working for me currently.

Comment (johno) [reply packages/gui/src/components/html/AttributeEditor.tsx]: Added that functionality, should be g2g now.

**Pull request merged**